File: JAVA/L3/P1/Class.java
package LABS.L3.P1;
import java.util.Scanner;

public class Class {
    private int classNumber;

    public Class(int classNumber) {
        this.classNumber = classNumber;
    }

    public int getClassNumber() {
        return classNumber;
    }

    public void setClassNumber(int classNumber) {
        this.classNumber = classNumber;
    }

    public void fill(Scanner scanner){
        classNumber = scanner.nextInt();
    }

    @Override
    public String toString() {
        return "Class:" +
                "Name= " + classNumber;
    }
}

File: C#/L5/Class.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace L5
{
    public class Class : InOut
    {
        private int classNumber;

        public Class()
        {
            classNumber = 0;
        }

        public Class(int classNumber)
        {
            this.classNumber = classNumber;
        }

        public int getClassNumber()
        {
            return classNumber;
        }

        public void setClassNumber(int classNumber)
        {
            this.classNumber = classNumber;
        }

        public void fill()
        {
            Console.WriteLine("\nClass\nclassNumber(int):\n");
            while (true)
            {
                try
                {
                    classNumber = int.Parse(Console.ReadLine());
                    break;
                }
                catch (Exception e)
                {
                    Console.WriteLine(e);
                }
            }
        }

        public void toString()
        {
            Console.Write("\nClass:\n" +
                    "Class_Number = " + classNumber + "\n");
        }
    }

}
